ksqldb-common/src/main/java/io/confluent/ksql/util/QueryApplicationId.java

```diff
@@ -51,7 +51,7 @@ public static String build(
     }
   }
 
-  private static String buildPrefix(
+  public static String buildPrefix(
```

Comment:
nit: rename this and the other to `buildInternalTopicPrefix` or similar to make it clear what exactly this returns the prefix for